src/notification.c

```diff
@@ -368,7 +368,6 @@ int notification_init(notification * n, int id)
 
         n->urls = notification_extract_markup_urls(&(n->body));
 
-        n->msg = string_replace_all("\\n", "\n", g_strdup(n->format));
         n->msg = notification_replace_format("%a", n->appname, n->msg,
```

Comment:
`n->msg` is NULL here leading to a segfault, I assume you didn't notice that the line above it also called `g_strdup` on the format to assign an initial value.